dlt/common/schema/schema.py

```diff
@@ -11,12 +11,19 @@
 from dlt.common.schema import utils
 from dlt.common.data_types import py_type_to_sc_type, coerce_value, TDataType
 from dlt.common.schema.typing import (COLUMN_HINTS, SCHEMA_ENGINE_VERSION, LOADS_TABLE_NAME, VERSION_TABLE_NAME, STATE_TABLE_NAME, TPartialTableSchema, TSchemaSettings, TSimpleRegex, TStoredSchema,
-                                      TSchemaTables, TTableSchema, TTableSchemaColumns, TColumnSchema, TColumnProp, TColumnHint, TTypeDetections)
+                                      TSchemaTables, TTableSchema, TTableSchemaColumns, TColumnSchema, TColumnProp, TColumnHint, TTypeDetections, TSchemaEvolutionModes, TSchemaEvolutionSettings)
 from dlt.common.schema.exceptions import (CannotCoerceColumnException, CannotCoerceNullException, InvalidSchemaName,
                                           ParentTableNotFoundException, SchemaCorruptedException)
 from dlt.common.validation import validate_dict
+from dlt.common.schema.exceptions import SchemaFrozenException
 
 
+DEFAULT_SCHEMA_EVOLUTION_MODES: TSchemaEvolutionModes = {
```

Comment:
1. do you have a preference for "Evolution" here? I like Schema Contract way more. ie DEFAULT_SCHEMA_CONTRACT_MODE
2. I'm a fan of simple settings that are strings (something like presets). can we have a few of them? they could be accepted in the same place where full TypedDict is accepted

Reply:
OK - I see (2) is implemented :)